Products/ZenUI3/browser/resources/js/zenoss/EventPanels.js

```diff
@@ -1490,11 +1490,20 @@
             this.refresh();
         },
         restoreURLState: function() {
-            var qs = window.location.search.replace(/^\?/, ''),
-            state = Ext.urlDecode(qs).state;
-            if (state) {
+            var qs = window.location.search.replace(/^\?/, ''), state, decState;
+            
+            if (qs) {
                 try {
```

Comment:
This `try` statement is now very large. it should be as minimal as possible. the reason it was present initially was to catch errors from `Zenoss.util.base64.decode`, so it needs to be minimized to that. This big `try` block could be especially frustrating since the `catch` block ignores the error, so whoever is debugging this would have a very hard time finding the problem.